### PR TITLE
Implements Navbar and Stories Page

### PR DESCRIPTION
**Pull request by taylorlew**

### CODE REVIEW

File: public/stylesheets/style_stories.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.card-deck {
+  display: grid;
+  grid-template-columns: auto auto auto;
```
Comment (KennyWibowo) [inline public/stylesheets/style_stories.css]: Breaks compatibility with IE, but who cares lol

File: routes/index.js [hunk 2]
```diff
@@ -20,6 +20,44 @@ function buildMessage(req) {
   }
 }
 
+function buildStoriesMessage(req) {
+  return {
```
Comment (KennyWibowo) [inline routes/index.js]: Could utilize DRY here - maybe create a function that combines this with the default buildMessage, or something?

File: views/navbar.hbs [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <a class="navbar-brand" href="/">Logo</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" href="#">About<span class="sr-only">(current)</span></a>
```
Comment (KennyWibowo) [inline views/navbar.hbs]: Why does this have "current"?

File: public/stylesheets/style_stories.css (hunk 1 above)
Comment (KennyWibowo) [reply public/stylesheets/style_stories.css]: Could potentially instead use bootstrap rows/columns

File: views/stories.hbs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<div class="card-deck">
+  {{#each stories}}
+  <div class="card">
+    <img class="card-img-top" src="..." alt="Card image cap">
+    <div class="card-body">
+      <h5 class="card-title">{{title}}</h5>
+      <p class="card-text">{{description}}</p>
+    </div>
+    <div class="card-footer">
+      <small class="text-muted">Last updated 3 mins ago</small>
```
Comment (KennyWibowo) [inline views/stories.hbs]: Maybe remove this, or move the title here

**Review by KennyWibowo — CHANGES_REQUESTED** [review]
Some suggestions to fix some stuff, but overall pretty good!

File: views/navbar.hbs (hunk 3 above)
Comment (KennyWibowo) [reply views/navbar.hbs]: could "current" be set dynamically based on the current webpage?

File: routes/index.js [hunk 5]
```diff
@@ -20,6 +20,44 @@ function buildMessage(req) {
   }
 }
 
+function buildStoriesMessage(req) {
```
Comment (KennyWibowo) [inline routes/index.js]: Add a comment for this function

File: routes/index.js [hunk 6]
```diff
@@ -20,6 +20,44 @@ function buildMessage(req) {
...
+  return {
+    title: 'Title',
+    style: 'stylesheets/style_stories.css',
+    user: req.user,
+    error: req.flash('error'),
+    warning: req.flash('warning'),
+    info: req.flash('info'),
+    success: req.flash('success'),
+    stories: [
```
Comment (KennyWibowo) [inline routes/index.js]: Add comment with TODO that links to [Back End] work: #9

**Review by KennyWibowo — CHANGES_REQUESTED** [review]

File: views/navbar.hbs (hunk 3 above)
Comment (taylorlew) [reply views/navbar.hbs]: Took <span class="sr-only">(current)</span> out. the sr-only means screen reader which is used for people who are visually impaired.  It makes it easier for them.

File: views/stories.hbs (hunk 4 above)
Comment (taylorlew) [reply views/stories.hbs]: Took out the description and just have the title as the footer.

**Comment by KennyWibowo** [comment]
Looks good to me, go ahead and merge :)

**Pull request merged**